providers/tests/system/google/cloud/dataprep/example_dataprep.py

```diff
@@ -313,6 +313,13 @@ def delete_connection(connection_id: str) -> None:
     # when "tearDown" task with trigger rule is part of the DAG
     list(dag.tasks) >> watcher()
 
+
+from dev.tests_common.test_utils.system_tests import get_test_run  # noqa: E402
+
+# Needed to run the example DAG with pytest (see: tests/system/README.md#run_via_pytest)
+test_run = get_test_run(dag)
```

Comment:
Something in the pre-commit made this a duplicated block -- it already exists a few lines below.

Reply:
resolved